ui/test/nuxeo-favorites-toggle-button.test.js

```diff
@@ -84,13 +92,19 @@ suite('nuxeo-favorites-toggle-button', () => {
         '{"entity-type": "document","uid": "1"}',
       ]);
     });
+    teardown(() => {
+      if (window.confirm.restore) {
+        window.confirm.restore();
+      }
+    });
 
     test('it should display the document as not favorite', () => {
       expect(element.favorite).to.be.false;
     });
 
-    test('toggle should add the document to favorites', async () => {
+    test('toggle should add the document to favorites when clicked on Ok', async () => {
```

Comment:
We should check that if we click cancel, it will not add the document to favorites.